### PR TITLE
Milestone 4 - Fetch API Data

### PR DESCRIPTION
**Pull request by cherelemma**
# Implementation Summary

In this Math Magicians project's 4th milestone, the following project requirements are implemented.

- Create a component for displaying a quote
- Render the quote component 

## General requirements checklist.

✅  No linters error.
✅  Followed the GitHub flow.
✅  Documented work professionally.

### CODE REVIEW

File: src/components/Fetch_API_Data.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+
+const FetchAPIData = () => {
+  const [data, setData] = useState([]);
```
Comment (BenMukebo) [inline src/components/Fetch_API_Data.js]: - [ ] This is the project requirements 👇🏻 Kindly make sure to create A `<Quotes>` component in which you will implement the fetchData  to the API with useEffect hook.

image

- 🤔 I think you only need to rename your fetch_API_Date.js file. 


```jsx
function Quotes() {
  const [quotes, setQuotes] = useState([])
  .....

  useEffect(() => {
    const fetchQuotes = async () => {
      setIsLoading(true)
      try {
        const res = await fetch(
          "https://api.api-ninjas.com/v1/quotes?category=happiness",
        .......
  
    fetchQuotes()
  }, [setQuotes])

  return (
    ....
  )
}

export default Quotes
```

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 import Calculator from './components/Calculator';
+import FetchAPIData from './components/Fetch_API_Data';
 
 function App() {
   return (
-    <Calculator />
+    <div className="wrapper">
+      <Calculator />
+      <FetchAPIData />
+    </div>
```
Comment (BenMukebo) [inline src/App.js]: - [ ] Then make sure the `<Quotes>` component is imported into the main component. Example:

```jsx
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @cherelemma,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @cherelemma , 

### Highlights ✨ 

- Thanks for making the changes 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**